server/database/constants.js

```diff
@@ -0,0 +1,407 @@
'Stained', 'Stale', 'Standard', 'Starchy', 'Stark', 'Starry', 'Steep', 'Sticky', 'Stiff', 'Stimulating', 'Stingy', 'Stormy', 'Straight', 'Strange', 'Steel', 'Strict', 'Strident', 'Striking', 'Striped', 'Strong', 'Studious', 'Stunning', 'Stupendous', 'Stupid', 'Sturdy', 'Stylish', 'Subdued', 'Submissive', 'Substantial', 'Subtle', 'Suburban', 'Sudden', 'Sugary', 'Sunny', 'Super', 'Superb', 'Superficial', 'Superior', 'Supportive', 'Sure-Footed', 'Surprised', 'Suspicious', 'Svelte', 'Sweaty', 'Sweet', 'Sweltering', 'Swift', 'Sympathetic', 'Tall', 'Talkative', 'Tame', 'Tan', 'Tangible', 'Tart', 'Tasty', 'Tattered', 'Taut', 'Tedious', 'Teeming', 'Tempting', 'Tender', 'Tense', 'Tepid', 'Terrible', 'Terrific', 'Testy', 'Thankful', 'That', 'These', 'Thick', 'Thin', 'Third', 'Thirsty', 'This', 'Thorough', 'Thorny', 'Those', 'Thoughtful', 'Threadbare', 'Thrifty', 'Thunderous', 'Tidy', 'Tight', 'Timely', 'Tinted', 'Tiny', 'Tired', 'Torn', 'Total', 'Tough', 'Traumatic', 'Treasured', 'Tremendous', 'Tragic', 'Trained', 'Tremendous', 'Triangular', 'Tricky', 'Trifling', 'Trim', 'Trivial', 'Troubled', 'True', 'Trusting', 'Trustworthy', 'Trusty', 'Truthful', 'Tubby', 'Turbulent', 'Twin', 'Ugly', 'Ultimate', 'Unacceptable', 'Unaware', 'Uncomfortable', 'Uncommon', 'Unconscious', 'Understated', 'Unequaled', 'Uneven', 'Unfinished', 'Unfit', 'Unfolded', 'Unfortunate', 'Unhappy', 'Unhealthy', 'Uniform', 'Unimportant', 'Unique', 'United', 'Unkempt', 'Unknown', 'Unlawful', 'Unlined', 'Unlucky', 'Unnatural', 'Unpleasant', 'Unrealistic', 'Unripe', 'Unruly', 'Unselfish', 'Unsightly', 'Unsteady', 'Unsung', 'Untidy', 'Untimely', 'Untried', 'Untrue', 'Unused', 'Unusual', 'Unwelcome', 'Unwieldy', 'Unwilling', 'Unwitting', 'Unwritten', 'Upbeat', 'Upright', 'Upset', 'Urban', 'Usable', 'Used', 'Useful', 'Useless', 'Utilized', 'Utter', 'Vacant', 'Vague', 'Vain', 'Valid', 'Valuable', 'Vapid', 'Variable', 'Vast', 'Velvety', 'Venerated', 'Vengeful', 'Verifiable', 'Vibrant', 'Vicious', 'Victorious', 'Vigilant', 'Vigorous', 'Villainous', 'Violet', 'Violent', 'Virtual', 'Virtuous', 'Visible', 'Vital', 'Vivacious', 'Vivid', 'Voluminous', 'Wan', 'Warlike', 'Warm', 'Warmhearted', 'Warped', 'Wary', 'Wasteful', 'Watchful', 'Waterlogged', 'Watery', 'Wavy', 'Wealthy', 'Weak', 'Weary', 'Webbed', 'Wee', 'Weekly', 'Weepy', 'Weighty', 'Weird', 'Welcome', 'Well-Documented', 'Well-Groomed', 'Well-Informed', 'Well-Lit', 'Well-Made', 'Well-Off', 'Well-To-Do', 'Well-Worn', 'Wet', 'Which', 'Whimsical', 'Whirlwind', 'Whispered', 'White', 'Whole', 'Whopping', 'Wicked', 'Wide', 'Wide-Eyed', 'Wiggly', 'Wild', 'Willing', 'Wilted', 'Winding', 'Windy', 'Winged', 'Wiry', 'Wise', 'Witty', 'Wobbly', 'Woeful', 'Wonderful', 'Wooden', 'Woozy', 'Wordy', 'Worldly', 'Worn', 'Worried', 'Worrisome', 'Worse', 'Worst', 'Worthless', 'Worthwhile', 'Worthy', 'Wrathful', 'Wretched', 'Writhing', 'Wrong', 'Wry', 'Yawning', 'Yearly', 'Yellow', 'Yellowish', 'Young', 'Youthful', 'Yummy', 'Zany', 'Zealous', 'Zesty', 'Zigzag', 'Rocky'];
```

Comment:
Can you format this for better readability?

Reply:
Yes. Thanks for the note.